ynr/apps/elections/uk/templates/official_documents/_post.html

```diff
@@ -1,33 +1,18 @@
-{% for doc_type, docs in official_documents %}
-  {% if doc_type.0 == 'Nomination paper' %}
+ {% if post_election.sopn %}
     <p>
-      {% if docs %}
-        {% for paper in docs %}
-          View the official Statement of Persons Nominated for this constituency:
-           <a href="{{ paper.get_absolute_url }}">{{ paper.document_type }}</a><br>
-        {% endfor %}
-        {% if user_can_upload_documents %}
-          <a href="{% url 'upload_document_view' election=election.slug post_id=post_data.id %}">
-          Upload an updated Statement of Persons Nominated</a>.
-        {% endif %}
-      {% else %}
-        We don't have nomination papers for {{ post_data.label }} yet.
-        {% if election == '2015' %}
-        Can you help us by <a href="https://docs.google.com/spreadsheets/d/1jvWaQSENnASZfGne1IWRbDATMH2NT2xutyPEbZ5Is-8/edit#gid=0">
-          adding them to this google spreadsheet</a>?
-        {% endif %}
-        {% if user_can_upload_documents %}
-          {% if election.slug == '2015' %}
-            Or, as you have permission to upload documents, you can
-            <a href="{% url 'upload_document_view' election=election.slug post_id=post_data.id %}">
-            upload it directly</a>.
-          {% else %}
-            Can you help us by
-            <a href="{% url 'upload_document_view' election=election.slug post_id=post_data.id %}">uploading
-            the PDF of the Statement of Persons Nominated?</a>
-          {% endif %}
-        {% endif %}
+        <a href="{{ post_election.sopn.get_absolute_url }}">
+          View the Statement of Persons Nominated for this ballot:
+        </a>
+
+      {% if user_can_upload_documents %}
+        <a href="{% url 'upload_document_view' election=election.slug post_id=post_data.id %}" class="button tiny">
+        Upload an updated Statement of Persons Nominated</a>.
```

Comment:
Also here. How about we remove the colon/full stop and put the button in a new block.

![screenshot at 2019-02-08 16-41-38](https://user-images.githubusercontent.com/6025893/52494768-badbd980-2bc6-11e9-8ca5-2eb0392e397f.png)